csrc/apis/java/classifier.cpp

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) OpenMMLab. All rights reserved.
+
+#include "classifier.h"
+
+#include "common.h"
+//#include <platform.h>
+//#include <benchmark.h>
+#include <jni.h>
+#include <android/log.h>
+#include <unistd.h>
+#include <stdio.h>
+
+// #include <opencv2/core/core.hpp>
+// #include <opencv2/imgproc/imgproc.hpp>
+
+namespace MMDeployJava {
+
+extern "C" {
+JNIEXPORT jobject JNICALL mmdeployClassifierCreateByPath(JNIEnv* env, jobject thiz,
+                                                         jstring modelPath, jstring deviceName,
+                                                         jint deviceID, jobject handlePointer) {
+  // handlePointer saves the value of mm_handle_t, because mm_handle_t is already a handle pointer.
+  int status{};
+  const char* model_path = env->GetStringUTFChars(modelPath, 0);
+  const char* device_name = env->GetStringUTFChars(deviceName, 0);
+  // handlePointer is a Java object which saves mm_handle_t classifier address.
+  jclass clazz = env->GetObjectClass(handlePointer);
+  jmethodID initMethod = env->GetMethodID(clazz, "<init>", "(Ljava/lang/String;J)V");
+  jfieldID id_address = env->GetFieldID(clazz, "address", "J");
+  mm_handle_t classifier = new mm_handle_t;
```

Comment:
`classifier` is supposed to be created inside `mmdeploy_classifier_create_by_path`. 
 Please refer to `demo/csrc/image_classification.cpp`